xcp_d/workflow/restingstate.py

```diff
@@ -106,22 +107,29 @@ def init_compute_alff_wf(
                          mem_gb=mem_gb,
                          name='alff_compt',
                          n_procs=omp_nthreads)
-    # create a node for the Nifti HTML
-    brain_plot = pe.Node(BrainPlot(),
-                         mem_gb=mem_gb,
-                         name='brain_plot',
-                         n_procs=omp_nthreads)
 
+    if not cifti:
+        alff_plot = pe.Node(Function(
+                            input_names=["output_path", "filename", "bold_file"],
+                            output_names=["output_path"],
+                            function=plot_alff_reho_volumetric),
+                            name="alff_nifti_plot")
+        alff_plot.inputs.output_path = 'alff.svg'
+        alff_plot.inputs.bold_file = bold_file
+    if cifti:
+        alff_plot = pe.Node(Function(
+                            input_names=["output_path", "filename", "bold_file"],
+                            output_names=["output_path"],
+                            function=plot_alff_reho_surface),
+                            name="alff_cifti_plot")
+        alff_plot.inputs.output_path = 'alff.svg'
+        alff_plot.inputs.bold_file = bold_file
```

Comment:
Can be simplified, since the only difference is the function.

```suggestion
    alff_plot = pe.Node(Function(
                        input_names=["output_path", "filename", "bold_file"],
                        output_names=["output_path"],
                        function= plot_alff_reho_surface  if cifti else plot_alff_reho_volumetric),
                        name="alff_plot")
    alff_plot.inputs.output_path = 'alff.svg'
    alff_plot.inputs.bold_file = bold_file
```